src/utils/beamer.ts

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import { BEAMER_ID } from './constants'
+
+export const loadBeamer = async (): Promise<void> => {
+  const APP_ID = BEAMER_ID
+  if (!APP_ID) {
+    console.error('[Beamer] - In order to use Beamer you need to add an appID')
+    return
+  }
+
+  window.beamer_config = {
+    product_id: APP_ID,
+    selector: 'whats-new-button',
+    display: 'left',
+    button: false,
+    bounce: false,
+  }
+
+  const beamerURL = 'https://app.getbeamer.com/js/beamer-embed.js'
+
+  const s = document.createElement('script')
```

Comment:
Good idea and also fixed a bug where the script was included multiple times! We should also refactor the intercom implementation since it happens there as well.